analyzer/src/main/kotlin/curation/FilePackageCurationProvider.kt

```diff
@@ -28,11 +29,27 @@ import org.ossreviewtoolkit.model.PackageCuration
 import org.ossreviewtoolkit.model.readValueOrDefault
 
 /**
- * A [PackageCurationProvider] that loads [PackageCuration]s from a single file. Supports all file formats specified
- * in [FileFormat].
+ * A [PackageCurationProvider] that loads [PackageCuration]s from all [curationFiles]. Supports all file formats
+ * specified in [FileFormat].
  */
-class FilePackageCurationProvider(curationFile: File) : PackageCurationProvider {
-    internal val packageCurations by lazy { curationFile.readValueOrDefault(emptyList<PackageCuration>()) }
+class FilePackageCurationProvider(curationFiles: Collection<File>) : PackageCurationProvider {
+    constructor(curationFile: File) : this(listOf(curationFile))
```

Comment:
BTW, an IMO nice pattern to combine both constructors is to use a signature like

    FilePackageCurationProvider(curationFile: File, vararg moreFiles: Collection<File>)

and then let the implementation merge `curationFile` and `moreFiles`. That way you ensure at compile time that at least one file is passed, but more can be passed as well. This is also the approach I'm following in my branch for upcoming `SimplePackageConfigurationProvider` changes.

Reply:
> That way you ensure at compile time that at least one file is passed

But this will take away the possibility to pass empty lists, which is being done in this case when no curation file is given.

My prefered option was to use `vararg`. But I get slapped on the fingers by detekt in the calling code:
`FilePackageCurationProvider(*curationFiles.toTypedArray()),`: https://detekt.github.io/detekt/performance.html#spreadoperator

Reply:
Then your *detekt*-pre-commit-check is somehow wrong, as we've [disabled that rule](https://github.com/oss-review-toolkit/ort/blob/bd2494938a82843932738780ce27707900976334/.detekt.yml#L41-L43) 😁